engine/Shopware/Controllers/Backend/Article.php

```diff
@@ -157,7 +157,25 @@ public function saveAction()
     {
         $data = $this->Request()->getParams();
         if ($this->Request()->has('id')) {
+            /** @var Article $article */
             $article = $this->getRepository()->find((int) $this->Request()->getParam('id'));
+            // Check whether the article has been modified in the meantime
+            try {
+                $data['changed'] = new \DateTime($data['changed']);
```

Comment:
I think you shouldn't set `$data['changed']` here and use a local variable instead, because that has less side effects - since `$data['changed']` is cleared anyway before saving.